src/main/java/com/mentoring/ecommerce/adapter/out/persistence/postgresql/product/ProductPostgreSQLAdapter.java

```diff
@@ -0,0 +1,28 @@
+package com.mentoring.ecommerce.adapter.out.persistence.postgresql.product;
+
+import com.mentoring.ecommerce.application.port.out.FindProductPort;
+import com.mentoring.ecommerce.application.port.out.SaveProductPort;
+import com.mentoring.ecommerce.application.port.out.UpdateProductPort;
+import com.mentoring.ecommerce.domain.Product;
+import lombok.RequiredArgsConstructor;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+// @PersistenceAdaper
```

Comment:
@Pelicer 
Should I create a `@PersistenceAdaper` annotation? 
What is its function?

Reply:
If I decide not to use it, must I add at least the `@Component` annotation?

Reply:
Yes